.spacemacs

```diff
@@ -62,6 +62,7 @@ values."
      syntax-checking
      version-control
      personal-config
+     xclipboard
```

Comment:
Do you know if this would work for Mac?

Reply:
I assume it will, but I tagged you and Phil to check it oot :grimacing:

Reply:
I'll try this branch on my machine and report back :)

Reply:
It does work on Mac! However, the `xclipboard` layer isn't yet in the Spacemacs master branch, which I was using. So to get this to work, I had to checkout the `develop` branch in my `~/.emacs.d`. We might need to mention this in the README?

Reply:
Ah, yes. I did mention it in the README update but I assumed everyone was using `develop`.

Reply:
So you did 😆 I should probably have reviewed the entire PR... Never mind, that's fine then.